kintone.d.ts

```diff
@@ -148,53 +195,60 @@ declare namespace kintone {
                 data: any,
                 callback: (resp: any) => void,
                 error: (err: any) => void
-            ) : void;
+            ): void;
 
             function setProxyConfig(
                 url: string,
                 method: string,
                 headers: any,
                 data: any,
                 callback: () => void
-            ) : void;
+            ): void;
         }
 
-        function getProxyCondig(url: string, method: string) : any;
-        
+        function getProxyCondig(
+            url: string,
+            method: string
+        ): any;
+
         function upload(
-            pluginId: any, 
+            pluginId: any,
             url: string,
             method: string,
             headers: any,
-            data: any,
-        ) : Promise<any>;
+            data: any
+        ): Promise<any>;
 
         function upload(
-            pluginId: any, 
+            pluginId: any,
             url: string,
             method: string,
             headers: any,
             data: any,
             callback: (resp: any) => void,
             error: (err: any) => void
-        ) : void;
+        ): void;
+    }
+
+    namespace portal {
+        function getContentSpaceElement(): Element | null;
```

Comment:
Should `getContentSpaceElement()` return a HTMLElement?

```suggestion
        function getContentSpaceElement(): HTMLElement | null;
```